frontend/src/metabase/entities/questions.js

```diff
@@ -102,7 +102,7 @@ const Questions = createEntity({
   forms,
 });
 
-function getIcon(question) {
+function getIcon(question, isSidebar) {
```

Comment:
Please, let's avoid passing UI details up to the entity layer. Can we make it return just the icon name and set the colors in the components themselves?